test/scenarios/ynETH.spec.sol

```diff
@@ -352,9 +352,9 @@ contract YnETHScenarioTest10 is IntegrationBaseTest, YnETHScenarioTest3 {
 		// The SelfDestructSender contract is created with the amountToSend and immediately self-destructs,
 		// sending its balance to the target address.
 		address(new SelfDestructSender{value: amountToSendViaSelfDestruct}(target));
-		
+
```

Comment:
Generally if you don't want to change a file just revert it back to the version on the base branch, so there's no random changes in it